packages/bpk-mixins/src/mixins/_margins.scss

```diff
@@ -0,0 +1,71 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// RTL aware leading margin
+///
+/// @param {string} $margin The desired leading margin.
+/// @param {boolean} $resetUppositeMargin [false] Whether to reset the opposite margin to 0.
+///
+/// @content
+///
+/// @example scss
+///   .selector {
+///     @include bpk-margin-leading($bpk-spacing-sm);
+///   }
+@mixin bpk-margin-leading($margin, $resetOppositeMargin:false) {
```

Comment:
Come to think of it we should have `resetOppositeMargin` be true by default imo

Reply:
Could we explore using [keyword arguments](http://thesassway.com/intermediate/leveraging-sass-mixins-for-cleaner-code) too? Especially with the boolean value for `resetOppositeMargin` I think the callsites might be a bit cryptic.

Reply:
I've defaulted it to `true`. Keyword arguments look cool, I didn't know that was a thing.